svc-api/handle/handlers.go

```diff
@@ -1134,6 +1134,24 @@ func SystemsMethodNotAllowed(ctx iris.Context) {
 	return
 }
 
+// SystemsMethodInvalidUrl holds builds reponse for the invalid url operation on Systems URLs and returns 404 error.
+func SystemsMethodInvalidUrl(ctx iris.Context) {
+	defer ctx.Next()
+	ctx.StatusCode(http.StatusNotFound)
+	errArgs := &errResponse.Args{
+		Code: errResponse.GeneralError,
+		ErrorArgs: []errResponse.ErrArgs{
+			errResponse.ErrArgs{
+				StatusMessage: errResponse.InvalidURL,
+				MessageArgs:   []interface{}{ctx.Request().Method},
```

Comment:
in lib utilities  InvalidURL accepting to argument and no argument  passed in caller please refer update the same